app/redux/thunks/CartThunks.js

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+import { setCart, addItemToCart, _removeItemFromCart } from "../actions";
+// temporary fetchCart thunk
+export function fetchCart() {
```

Comment:
pass a userId into the function. the route will call `/api/users/${ :userId }/cart`